src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -188,6 +188,19 @@ std::string postUpdateContext
 
     cleanPayload = xmlPayloadClean(out.c_str(), "<updateContextResponse>");
 
+    if ((cleanPayload == NULL) || (cleanPayload[0] == 0))
+    {
+      UpdateContextResponse ucrs;
+
+      // FIXME P1: What StatusCode should be used here?
```

Comment:
ok, sure.  It is a strange situation. It is a 500, but not in the broker, but in the context provider.
A user might not know about context providers and see (orion + context providers) as a black box.
This reasoning led me to return a 500 (+ the FIXME about which code to use :-))